internal/pkg/template/templates/workloads/partials/cf/sidecars.yml

```diff
@@ -1,6 +1,10 @@
 {{if .LogConfig}}
 - Name: firelens_log_router
   Image: {{ .LogConfig.Image }}
+  Environment:
+{{/* "$" denotes the parent WorkloadOpts, whereas "." is the individual container. */}}
+{{include "envvars-common" $ | indent 2}}
+{{include "envvars-container" . | indent 2}}
```

Comment:
I'm confused 🤔 does this work?

"." at l.46 refers to `$sidecar` afaict whereas here it would be the entire WorkloadOpts. I might be misunderstanding something though

Instead do we have to update `LogConfig` to have `Variables` and `Secrets` fields?

Reply:
Oh we need to fix this. Right now we are just using the env var for the main container. We need to define them in logconfig.

Reply:
ok done boss